hci_cyw43439.go

```diff
@@ -0,0 +1,27 @@
+//go:build cyw43439
+
+package bluetooth
+
+const (
+	ogfVendor = 0x3f
+
+	ocfSetBTMACAddr = 0x0001
+)
+
+func (a *Adapter) SetBdAddr(address Address) error {
```

Comment:
Not a fan of the naming-  I'd prefer `SetAddress` since the type it receives is an `Address`

Reply:
Sure I can rename it. The function to get the Bluetooth address was named GetBdAddr which is where the name comes from